common-test/src/main/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -470,4 +474,35 @@ private KubeClusterResource clusterResource() {
         }
         return clusterResource;
     }
+
+    private Statement withTopic(Annotatable element, Statement statement) {
+        Statement last = statement;
+        for (Topic cluster : annotations(element, Topic.class)) {
+            String yaml = getContent(new File(TOPIC_CM), node -> {
+                JsonNode metadata = node.get("metadata");
+                ((ObjectNode) metadata).put("name", String.valueOf(cluster.name()));
```

Comment:
No need to use String.valueOf() for cluster.name() because method name() already returns String value.

Reply:
And I think that you should use the topic name for this key. WDYT?